docs/integrations/sources/incident-io.md

```diff
@@ -0,0 +1,59 @@
+# Incident.io
+
+This is the setup guide for the Incident.io source connector which ingests data from the incident API.
```

Comment:
This sentence is not clear enough. What are the two most useful streams? 

Calling this a guide is not necessary. It's pretty obvious it's a guide from where we render this page.